integration/test/Test/Teams.hs

```diff
@@ -123,7 +127,11 @@ testInvitePersonalUserToTeam = do
     checkListInvitations :: Value -> String -> String -> App ()
     checkListInvitations owner tid email = do
       newUserEmail <- randomEmail
-      void $ postInvitation owner (PostInvitation (Just newUserEmail) Nothing) >>= assertSuccess
+      inv <- postInvitation owner (PostInvitation (Just newUserEmail) Nothing) >>= assertSuccess
+      code <- I.getInvitationCode owner inv >>= getJSON 200 >>= (%. "code") & asString
```

Comment:
This unfortunately fails on CI: https://concourse.ops.zinfra.io/builds/72288342#L66ee9972:736